projects/dotcms/src/lib/api/DotApiContentType.ts

```diff
@@ -13,25 +13,64 @@ export class DotApiContentType {
     }
 
     get(contentTypeId): Promise<DotCMSContentType> {
-        return this.dotCMSHttpClient.request({
-            url: `/api/v1/contenttype/id/${contentTypeId}`
-        })
-        .then(async (response: Response) => {
-            if (response.status === 200) {
-                const data = await response.json();
-                return data.entity;
-            }
-
-            throw <DotCMSError>{
-                message: await response.text(),
-                status: response.status
-            };
-        });
+        return this.dotCMSHttpClient
+            .request({
+                url: `/api/v1/contenttype/id/${contentTypeId}`
+            })
+            .then(async (response: Response) => {
+                if (response.status === 200) {
+                    const data = await response.json();
+                    return data.entity;
+                }
+
+                throw <DotCMSError>{
+                    message: await response.text(),
+                    status: response.status
+                };
+            });
     }
 
     getFields(contentTypeId): Promise<DotCMSContentTypeField[]> {
         return this.get(contentTypeId).then((contentType: DotCMSContentType) => {
             return contentType.fields;
         });
     }
+
+    save<Content extends DotCMSContent>(params: Content): Promise<Response> {
+        return this.dotCMSHttpClient
+            .request({
+                url: `/api/content/save/1`,
+                method: 'POST',
+                body: JSON.stringify(params)
+            })
+            .then(async (response: Response) => {
+                if (response.status === 200) {
+                    return null;
```

Comment:
it is weird return null when success, why not return response?

Reply:
agree

Reply:
the Endpoint does not return anything

Reply:
Should we return the response?